gpu-viewer.in

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+cd @prefix@/@datadir@/gpu-viewer/Files
```

Comment:
Definitely do not do string concatenation here! :)

datadir can, since Meson 0.62.0, be an absolute path. It will be on NixOS.